src/App.tsx

```diff
@@ -1,24 +1,146 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Eror } from './components/ERROR';
```

Comment:
The component name 'Eror' should be changed to 'ErrorComponent' for better readability and consistency.